data.py

```diff
@@ -107,6 +107,13 @@ def update_bottle(name, homie_fb_id):
     except:
         raise
 
+def get_bottle(homie_fb_id):
+    try:
+        bottle_entry = execute_statement("SELECT curr_bottle_id from homies where homie_fb_id = %s;", args=(homie_fb_id), ret=True)
```

Comment:
(homie_fb_id)

This is an expression and not a tuple, which psycopg requires its arguments to be passed in as

Reply:
Resolved